tutorials/katacoda/thanos/x-secret-rawkode/intro.md

```diff
@@ -0,0 +1,8 @@
+# Demo Resources
+
+Let's go!
+
+### Feedback
+
```

Comment:
Do we still need to keep this secret demo?

Reply:
Nvm seems this is removed already.

Reply:
Yes (: I moved it, thanks for looking!